source/New_Values.py

```diff
@@ -0,0 +1,15 @@
+from CardGames import *
```

Comment:
Maybe next time add comments for whoever is reviewing :)

Reply:
Yeah, forgot to add comments. Will try to remember to put some next time.